docs-2.0/k8s-operator/4.cluster-administration/4.7.security/4.7.1.enable-mtls.md

```diff
@@ -493,34 +502,92 @@ Details about these fields are as follows:
+As NebulaGraph Operator needs to communicate with the Meta and Storage services, you must configure `spec.sslCerts` to set certificates for it when enabling mTLS between services.
 
-The `spec.sslCerts` field specifies the encrypted certificates for NebulaGraph Operator and the [nebula-agent](https://github.com/vesoft-inc/nebula-agent) client (if you do not use the default nebula-agent image in Operator).
+The `spec.sslCerts` field specifies the encryption certificates for NebulaGraph Operator and [nebula-agent](https://github.com/vesoft-inc/nebula-agent) (if you do not use the default nebula-agent image).
 
-```yaml
-spec:
-  sslCerts:
-    clientSecret: "client-cert" 
-    clientCert: "tls.crt"  # The key name of the certificate in the client certificate Secret, default is "tls.crt".
-    clientKey: "tls.key"   # The key name of the private key in the client certificate Secret, default is "tls.key".
-    caSecret: "ca-cert" # The name of the CA certificate Secret.
-    caCert: "tls.crt"  # The key name of the certificate in the CA certificate Secret, default is "tls.crt".
-```
+You can create a Secret to pass in the certificates or configure the environment variables of NebulaGraph Operator to obtain local certificates.
 
-!!! caution
-    Ensure that the keys for the certificates and private keys in the created Secret match the values of the `clientCert`, `clientKey`, and `caCert` fields in `spec.sslCerts`. If you've created a Secret of the TLS type, the default key names are `tls.crt` and `tls.key` for the certificate and private key. In this case, you'll need to manually configure `caCert` field with `ca.crt`.
+- In the [chart configuration file of NebulaGraph Operator](../../3.operator-management/3.1.customize-installation.md), [update](../../3.operator-management/3.2.update-operator.md) the `controllerManager.env` value to set the certificate environment variables.
 
-The `insecureSkipVerify` field is used to determine whether the client will verify the server's certificate chain and hostname. The default value is `false`. In production environments, it is recommended to set this to `false` to ensure the security of communication. If set to `true`, the client will not verify the server's certificate chain and hostname.
+  The Nebula-Operator chart allows configuration of the init container, sidecar container, volumes, and volumeMounts within the controller-manager Pod. It accesses the certificates generated by the sidecar-container through the environment variables `CA_CERT_PATH`, `CLIENT_CERT_PATH`, and `CLIENT_KEY_PATH`. To generate certificates, you need to place the sidecar-container and the controller-manager container in the same Pod and share the certificate directory credentials. The controller-manager then reads the client-side certificates from the environment variables and establishes a mutual TLS (mTLS) connection with the Meta and Storage services. Example:
```

Comment:
Please specify the following configurations are for operator.

Reply:
And state that once environment variables are set in **operator**, all the secrets in `sslCerts` in the cluster configuration will be ignored.

Reply:
done